masterlist.yaml

```
@@ -14611,13 +14616,94 @@ plugins:
 
   - name: 'RUSTIC SOULGEMS - (Sorted|Unsorted)( - ESL)?( - GIST)?\.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5785/' ]
+    tag: [ Graphics, ObjectBounds ]
     inc:
       - name: 'MLPSoulGems.esp'
         display: 'MultiLayer Parallax Soul Gems'
     msg:
       - <<: *useOnlyOneX
         condition: 'many("RUSTIC SOULGEMS - (Sorted|Unsorted)( - ESL)?( - GIST)?\.esp")'
         subs: [ 'RUSTIC SOULGEMS' ]
+  - name: RUSTIC SOULGEMS - Sorted.esp
```

Missing apostrophe on file names.

```suggestion
  - name: 'RUSTIC SOULGEMS - Sorted.esp'
```